KubeArmor/enforcer/bpflsm/enforcer.go

```diff
@@ -300,6 +300,10 @@ func (be *BPFEnforcer) TraceEvents() {
 			continue
 		}
 
+		readLink := false
+		if len(event.Data.Source) == 0 {
```

Comment:
```suggestion
		if len(string(bytes.Trim(event.Data.Source[:], "\x00"))) == 0 {
```
It might not be empty but filled with garbage value